guide/03-the-gis/accessing-and-creating-content.ipynb

```diff
@@ -2332,7 +2332,7 @@
    "source": [
     "**Note**: here, we used the `add()` method to add the `FeatureCollection` object in memory as an item on the GIS. Notice we used the `text` property to pass the JSON representation of the feature collection and the `type` property to indicate the item type. You can use the same method to publish web maps and web scenes.\n",
     "\n",
-    "The [sample notebooks for content publishers](https://developers.arcgis.com/python/sample-notebooks/publishing-web-maps-and-web-scenes/) show how the concepts discussed here can be used to publish web maps and scenes to your GIS"
+    "The [sample notebooks for content publishers](https://developers.arcgis.com/python/lastest/samples/publishing-web-maps-and-web-scenes/) show how the concepts discussed here can be used to publish web maps and scenes to your GIS"
```

Comment:
Looks like this link is still not working. I noticed that after clicking the link, there is another `latest` added to the url 😕 @jyaistMap , how do we handle this kind of urls?
![image](https://github.com/user-attachments/assets/d6407390-048a-456d-a985-12d22114a3ec)